apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/patientSearchByIdWithIncludeIdentifiers.json

```diff
@@ -165,6 +165,24 @@
         "period" : {
           "start" : "2020-07-30"
         }
+      }, {
+        "type" : {
+          "coding" : [ {
+            "extension" : [ {
+              "url" : "https://bluebutton.cms.gov/resources/codesystem/identifier-currency",
+              "valueCoding" : {
+                "system" : "https://bluebutton.cms.gov/resources/codesystem/identifier-currency",
+                "code" : "historic",
+                "display" : "Historic"
+              }
+            } ],
+            "system" : "http://terminology.hl7.org/CodeSystem/v2-0203",
+            "code" : "MC",
+            "display" : "Patient's Medicare number"
+          } ]
+        },
+        "system" : "http://hl7.org/fhir/sid/us-mbi",
+        "value" : "543217066"
```

Comment:
Wondering why the v2 responses changed here; the other table's entries were added to the file for bene_history, but it _shouldnt_ have changed the output, since it was being left joined before this PR when creating the response.

Reply:
So that is an interesting thing about some of the V2 tests; when checking Identifier entries, our verification should be agnostic with respect to order of entries. It is not. So had to re-gen a couple of the JSON results since they were slightly different due to order.

Reply:
Not quite sure where you are coming from r.e. 'added records'. Yes,
I added data because sample-a-medicarebeneficiaryidhistory.txt file(s)
were removed; their data was added to the sample-a-beneficiaryhistory.txt
file. Also, since original code loaded in MEDICARE_BENEFICIARYID_HISTORY
records and then BENEFICIARIES_HISTORY records, by defintiion, the order
of processing/rendering of JSON will change. Hence the need to tweak the
'golden' JSON output(s).

Here is what the current sample-a-beneficiaryhistory.txt looked like:

DML_IND BENE_ID BENE_BIRTH_DT BENE_CRNT_HICN MBI_NUM EFCTV_BGN_DT
INSERT  567834  17-Mar-79     543217066Z     3456689 17-Mar-90
INSERT  567834  17-Mar-80     543217066T     3456789 17-Mar-90
INSERT  567834  17-Mar-80     543217066T     3456789 17-Mar-90

Here is what the updated sample-a-beneficiaryhistory.txt looks like:

DML_IND BENE_ID BENE_BIRTH_DT BENE_CRNT_HICN MBI_NUM EFCTV_BGN_DT
INSERT  567834  17-Mar-79     543217066Z     3456689 17-Mar-90
INSERT  567834  17-Mar-80     543217066T     3456789 17-Mar-90
INSERT  567834  17-Mar-80     543217066T     3456789 17-Mar-90
INSERT  567834  17-Mar-80     543217077T 9AB2WW3GR44 15-Nov-87
INSERT  567834  17-Mar-80     543217055T   543217066 16-Jan-87

I took records from the sample-a-medicarebeneficiaryidhistory.txt file
and the sample-a-medicarebeneficiaryidhistory-extra.txt file and added
them to the sample-a-beneficiaryhistory.txt file as needed.

So as far as unique MBI_NUM(s) we have:
3456689, 3456789, 9AB2WW3GR44, 543217066

3456789 is also the current MBI_NUM and therefore does not count as 'historical';
record(s) exist in the beneficiaryhistory because the ETL rule is that if ANY of
7-8 fields differ (see RifLoader.isBeneficiaryHistoryEqual) a new history record
is created. For example, an update to a bene's DOB, would create a new history
record, but their MBI_NUM has not changed.


9AB2WW3GR44, 543217066 came out of the 'old' medicarebeneficiaryidhistory data
and therefore the order in which they would be rendered will now differ from
what the captured JSON file(s) will show; thus the need to re-capature JSON
payloads for the comparisons.

If I am missing something, please let me know. Thanks.

Reply:
Thanks, you covered my confusion; there was a new "historical" number added because the other table tests had a different "current" number, so when the tables were combined created a new historical number out of the previous table's current number.